.gitignore

```diff
@@ -6,6 +6,7 @@ docs/
 *.autosave
 bin/macosx64/Sparkle.framework
 bin/macosx64/borg
```

Comment:
oops. i recently killed that subdir because the vorta.spec works more easily in `bin/` (not `bin/macosx64`).
is it required by sparkle?

Reply:
Not required. I just keep it there. This entry is redundant anyways due to the `bin/*` below.